optimum_transformers/pipelines/fill_mask.py

```diff
@@ -1,222 +1,20 @@
-from typing import Dict
-
-import numpy as np
-
-from transformers.file_utils import add_end_docstrings, is_tf_available, is_torch_available
 from transformers.modeling_outputs import MaskedLMOutput
 from transformers.utils import logging
-from .base import PIPELINE_INIT_ARGS, GenericTensor, Pipeline, PipelineException
-
-if is_tf_available():
-    import tensorflow as tf
-
-if is_torch_available():
-    import torch
+from .base import _forward_onnx, _warmup_onnx_graph
+from transformers import FillMaskPipeline
+import torch
 
 logger = logging.get_logger(__name__)
```

Comment:
I saw some class with the logger and some others without it. Is it a legacy from HF Transformers package or is there a logger in this project ?

Reply:
Exists in this project
Will update and rerequest review